label-inspector/label_inspector.ipynb

```diff
@@ -796,43 +805,609 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Other Tips"
+    "## 6. Deploy trained model and perform real time inference"
```

Comment:
Clarify the rest of this notebook is totally optional?

I think many users may only want the label analysis

Reply:
I would first have an overarching section on "Deploy model to make predictions on new data"

and then subsections:

"Perform batch inference"

"Perform real time inference"

What made you choose to present real-time first? Isn't that more complex?

Reply:
No particular reason, the skeleton template for sample notebooks (+ other sample notebooks such as the AG one) presents real time first so was just mimicking the structure.

I can flip the order around if you think it makes more sense!

Reply:
ya let's switch order if it's not too much trouble. I imagine batch inference is easier to interact with for most folks (since they don't need to manage the endpoint), whereas real-time is more just for those whose needs aren't met by batch inference.